src/invoice2data/extract/invoice_template.py

```diff
@@ -121,9 +127,10 @@ def matches_input(self, optimized_str: str) -> bool:
             return False
 
     def parse_number(self, value):
-        assert (
-            value.count(self.options["decimal_separator"]) < 2
-        ), "Decimal separator cannot be present several times"
+        assert value.count(self.options["decimal_separator"]) < 2, (
+            "Error in Template %s \nDecimal separator cannot be present several times"
```

Comment:
The other 7 asserts don't use `\n` in messages, I'd suggest dropping `\n` from this one for compatibility.